tests/test_get_by_label.py

```diff
@@ -33,29 +39,27 @@ def test_get_by_label_all_onto() -> None:
 
     testonto = get_ontology("http://domain_ontology/new_ontology")
     testonto.new_entity("Class", owlready2.Thing)
-    assert testonto._label_annotations == None
     assert testonto.get_by_label_all("*") == {testonto.Class}
     testonto.new_annotation_property(
         "SpecialAnnotation", owlready2.AnnotationProperty
     )
     testonto.Class.SpecialAnnotation.append("This is a comment")
-    testonto.set_default_label_annotations()
 
     testonto.new_entity("Klasse", testonto.Class)
 
-    assert testonto.Klasse.prefLabel == ["Klasse"]
-
-    testonto.Klasse.altLabel = "Class2"
+    # THESE NOW FAILS, SINCE prefLabel AND altLabel ARE NOT DEFINED in testonto
```

Comment:
But what do we use instead if we make an ontology from scratch and then want to add the default label annotations? That is what this test was checking